plugin.video.nhlgcl/addon.xml

```diff
@@ -1,26 +1,23 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.nhlgcl" name="NHL TV™" version="2017.2.24" provider-name="eracknaphobia">
+<addon id="plugin.video.nhlgcl" name="NHL TV™" version="2017.10.3" provider-name="eracknaphobia">
   <requires>
-   <import addon="xbmc.python" version="2.24.0"/>   
-   <import addon="script.module.simplejson" version="3.3.0"/>  
-   <import addon="script.module.pytz" version="2014.2" />   
-   <import addon="script.module.beautifulsoup4" version="4.3.1" /> 
-   <import addon="script.module.pil" version="1.1.7" /> 
+   <import addon="xbmc.python" version="2.24.0"/>
+   <import addon="script.module.simplejson" version="3.3.0"/>
```

Comment:
Json should have been included with jarvis, don't think this module is necessary. :)

Reply:
I'll remove unnecessary module import